src/RiotAPILibraries.ts

```diff
@@ -84,76 +84,95 @@ export default class RiotAPILibraries {
-                message.channel.send(this.settings.githubLibraries.noLanguage + language);
-                return;
-            }
+                const libraryList = await response.json();
+                if (!Array.isArray(libraryList) || libraryList.length === 0 || !libraryList[0].sha) {
+                    message.channel.send(this.settings.githubLibraries.noLanguage + language);
+                    return;
+                }
 
-            const editMessagePromise = message.channel.send(`Found the list of libraries for ${language}, listing ${libraryList.length} libraries, this post will be edited with the result.`);
+                const editMessagePromise = message.channel.send(`Found the list of libraries for ${language}, listing ${libraryList.length} libraries, this post will be edited with the result.`);
 
-            const promises = libraryList.map(lib => this.describeAPILibrary(lib));
-            const libraryDescriptions = (await Promise.all(promises))
-                .filter(l => l.valid && l.library) // Only valid ones
-                .sort((a, b) => b.stars - a.stars); // Sort by stars
+                const promises = libraryList.map(lib => this.describeAPILibrary(lib));
+                const libraryDescriptions = (await Promise.all(promises))
+                    .filter(l => l.valid && l.library) // Only valid ones
+                    .sort((a, b) => b.stars - a.stars); // Sort by stars
 
-            const embed = new Discord.RichEmbed({ title: `List of libraries for ${language}:` });
-            for (const desc of libraryDescriptions) {
-                if (!desc.library) {
-                    // https://github.com/Microsoft/TypeScript/issues/18562
-                    continue;
+                const embed = new Discord.RichEmbed({ title: `List of libraries for ${language}:` });
+                for (const desc of libraryDescriptions) {
+                    if (!desc.library) {
+                        // https://github.com/Microsoft/TypeScript/issues/18562
+                        continue;
+                    }
+                    embed.addField(`${desc.library.repo} (★ ${desc.stars ? desc.stars : "0"})`, `${desc.library.description ? desc.library.description + "\n" : " "}${desc.links.join(", ")}`);
                 }
-                embed.addField(`${desc.library.repo} (★ ${desc.stars ? desc.stars : "0"})`, `${desc.library.description ? desc.library.description + "\n" : " "}${desc.links.join(", ")}`);
-            }
 
-            let editMessage = await editMessagePromise;
-            if (Array.isArray(editMessage)) { editMessage = editMessage[0]; }
-            editMessage.edit({ embed });
+                let editMessage = await editMessagePromise;
+                if (Array.isArray(editMessage)) { editMessage = editMessage[0]; }
+                editMessage.edit({ embed });
+            } catch (error) {
+                console.error(`Error occurred fetching from github: ${error}`);
```

Comment:
Send something to the user? There might be a stale message where botty promises to edit something

Reply:
Fixed, github wont show as outdated for some reason..